src/passes/CoalesceLocals.cpp

```diff
@@ -595,8 +595,14 @@ void CoalesceLocalsWithLearning::pickIndices(std::vector<Index>& indices) {
 
 // declare passes
 
-static RegisterPass<CoalesceLocals> registerPass1("coalesce-locals", "reduce # of locals by coalescing");
```

Comment:
If we are removing all the `RegisterPass` calls, can we now just delete its implementation? I don't see that in this PR.